### DISCUSSION TITLE
Fake PS3 controller

### DISCUSSION THREAD

**Discussion by digiallmemory** [darthcloud/BlueRetro#840] [General]
Is there any way to sync unofficial ps3 controller with blueretro ??

**Comment by darthcloud** [comment]
Those things are very broken. Currently it's not possible. See #9

**Comment by Tredes** [comment]
Hi. There are some generic PS3-type controllers of Chinese origin that alternate their pairing mode by holding down the PS button. The flashing occurs in a single LED, it is fast and manages to link with Blueretro, it is an unusual controller
